syntax = "proto3";

import "shared.proto";

message HeaderRequest {
  optional string version = 1;
}

message HeaderReply {
  optional Status status = 1;
  optional string message = 2;
}
